### PR TITLE
feat(table-widget): apply column-width fields to Table widget 

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA
image

### CODE REVIEW

File: apps/web/src/common/modules/widgets/_constants/widget-field-constant.ts [hunk 1]
```diff
@@ -96,3 +96,5 @@ export const PIE_CHART_SERIES_LABEL_POSITION = {
     inner: 'inner',
     outer: 'outer',
 } as const;
+
+export const DEFAULT_MINIMUM_WIDTH = 120;
```
Comment (yuda110) [inline apps/web/src/common/modules/widgets/_constants/widget-field-constant.ts:100]: suggestion: How about `TABLE_DEFAULT_MINIMUM_WIDTH`?

File: apps/web/src/common/modules/widgets/_widgets/table/_component/WidgetDataTable.vue [hunk 2]
```diff
@@ -276,6 +281,31 @@ const getHeatmapColorStyle = (item: TableDataItem, field: TableWidgetField) => {
...
+    let customWidth: string|number|undefined = props.customTableColumnWidthInfo?.value?.find((item) => item.fieldKey === field?.name)?.width;
+    if (props.fieldType === 'staticField') {
+        if (field?.fieldInfo?.additionalType === 'comparison') customWidth = props.customTableColumnWidthInfo?.value?.find((item) => item.fieldKey === field?.name.replace('comparison_', ''))?.width;
+    } else if (props.fieldType === 'dynamicField') {
+        if (field?.fieldInfo?.type === 'dataField') customWidth = props.customTableColumnWidthInfo?.value?.find((item) => item.fieldKey === props.dataField as string)?.width;
+    }
+    const fixedWidth = props.tableColumnWidthInfo?.fixedWidth;
+    const calculatedWidth = customWidth || fixedWidth;
+    return calculatedWidth ? `${calculatedWidth}px` : undefined;
+};
```
Comment (yuda110) [inline apps/web/src/common/modules/widgets/_widgets/table/_component/WidgetDataTable.vue:307]: praise: wow...

**Review by yuda110 — APPROVED** [review]
good...

File: apps/web/src/common/modules/widgets/_constants/widget-field-constant.ts (hunk 1 above)
Comment (piggggggggy) [reply apps/web/src/common/modules/widgets/_constants/widget-field-constant.ts:100]: 아 그렇네

**Pull request merged**